Project.toml

```diff
@@ -58,7 +58,7 @@ CommonDataModel = "0.2.3, 0.3"
 ConstructionBase = "1"
 CoordinateTransformations = "0.6.2"
 DataFrames = "1"
-DimensionalData = "0.28.2"
+DimensionalData = "0.29"
```

Comment:
This may need other changes, lets do that elsewhere and when we have to break things